packages/create-wagmi/tsup.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from 'tsup'
+
+import { dependencies } from './package.json'
+
+export default defineConfig({
+  bundle: true,
+  clean: true,
+  entry: ['src/index.ts'],
+  format: ['esm'],
+  external: Object.keys(dependencies),
+  platform: 'node',
```

Comment:
nitpick: can we alphabetize these?

```suggestion
  bundle: true,
  clean: true,
  entry: ['src/index.ts'],
  external: Object.keys(dependencies),
  format: ['esm'],
  platform: 'node',
```